tools/wptrunner/wptrunner/browsers/chrome.py

```diff
@@ -194,17 +205,96 @@ def env_options():
 def update_properties():
     return (["debug", "os", "processor"], {"os": ["version"], "processor": ["bits"]})
 
-
 class ChromeBrowser(WebDriverBrowser):
+    def __init__(self,
+                 logger: StructuredLogger,
+                 **kwargs: Any):
+        super().__init__(logger, **kwargs)
+        self._actual_port = None
+
     def restart_on_test_type_change(self, new_test_type: str, old_test_type: str) -> bool:
         # Restart the test runner when switch from/to wdspec tests. Wdspec test
         # is using a different protocol class so a restart is always needed.
         if "wdspec" in [old_test_type, new_test_type]:
             return True
         return False
 
+    def create_output_handler(self, cmd: List[str]) -> OutputHandler:
+        """Return an instance of the class used to handle application output.
```

Comment:
Seems like this docstring got copied into the subclass, which it probably doesn't need to be.

Reply:
Thank you James,
As for the failing infrastructure/tests. I see that the failure happens while running WPT against geckodriver.
This could not be affected by my fix.
What can we do in this case?